pretext/utils.py

```diff
@@ -1,5 +1,5 @@
 from asyncio import subprocess
-import os
+import os, random_package
```

Comment:
I guess this is valid with Black? The PR fails for other reasons (package DNE) obviously but I expected this to get flagged.